plugins/modules/ssh_config.py

```diff
@@ -160,24 +161,16 @@
 '''
 
 import os
-import traceback
 
 from copy import deepcopy
 
-STORM_IMP_ERR = None
-try:
-    from storm.parsers.ssh_config_parser import ConfigParser
-    HAS_STORM = True
-except ImportError:
-    HAS_STORM = False
-    STORM_IMP_ERR = traceback.format_exc()
-
 from ansible.module_utils.basic import AnsibleModule, missing_required_lib
 from ansible.module_utils.common.text.converters import to_native
+from ansible_collections.community.general.plugins.module_utils._stormssh import ConfigParser, HAS_PARAMIKO, PARAMIKO_IMPORT_ERROR
 from ansible_collections.community.general.plugins.module_utils.ssh import determine_config_file
 
 
-class SSHConfig():
+class SSHConfig:
```

Comment:
Not that it makes much a difference, but (almost) all other code in this repo does it this way, so:
```suggestion
class SSHConfig(object):
```